Frameworks/UIKit/StarboardXaml/ApplicationCompositor.cpp

```diff
@@ -39,14 +39,8 @@ void InitializeApp() {
     initialized = true;
 
     // Set our writable and temp folders
-    auto pathData = Windows::Storage::ApplicationData::Current->LocalFolder->Path;
-    IwSetWritableFolder(pathData->Data());
-
-    char writableFolder[2048];
-    size_t outLen;
-    auto tempPathData = Windows::Storage::ApplicationData::Current->TemporaryFolder->Path;
-    wcstombs_s(&outLen, writableFolder, tempPathData->Data(), sizeof(writableFolder) - 1);
-    SetTemporaryFolder(writableFolder);
+    IwSetWritableFolder(Windows::Storage::ApplicationData::Current->LocalFolder->Path->Data());
+    SetTemporaryFolder(Windows::Storage::ApplicationData::Current->TemporaryFolder->Path->Data());
```

Comment:
? the first one is required because starboard is removed from the application layer.  The second is rather bizarre, but NSPathUtilties depends on it.  Sure, NSPathUtiltities can do this instead.